template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/RazorPagesWeb-CSharp/Pages/Account/Manage/TwoFactorAuthentication.cshtml

```diff
@@ -5,44 +5,32 @@
 }
 
 <h4>@ViewData["Title"]</h4>
-@if (Model.HasAny2faProviders)
+@if (Model.HasAny2faProviders && Model.Is2faEnabled)
```

Comment:
Yeah but I think we are in an in between state now, where the 2fa flow basically is very targetted towards the authenticator.  They will likely still need to do roughly what we did to enable authenticator to make SMS/email work again (unlike previous templates which had more of the SMS/email stuff baked in to attempt to make it more straightforward to drop in additional TFA providers).  I thought we eliminated most of the code/comments that were there to try and make it easy to add back other 2FA providers?